packages/lib-ethers/utils/deploy.ts

```diff
@@ -20,6 +20,12 @@ export const setSilent = (s: boolean): void => {
   silent = s;
 };
 
+// TODO: Set when LED Oracle is deployed
```

Comment:
These might be more appropriate in the `hardhat.config.ts` file similar to the existing `oracleAddresses`.

Reply:
Moved to `hardhat.config.ts`